core/src/main/java/org/apache/carbondata/core/scan/result/BlockletScannedResult.java

```diff
@@ -153,6 +154,8 @@
 
   private ReusableDataBuffer[] measureReusableBuffer;
 
+  int startIndex = 0;
```

Comment:
add a comment what is it and why it is required

Reply:
done

Reply:
done

Reply:
done